### DISCUSSION TITLE
Metadata tabs

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#2041] [Q&A] [answered]
Hello
In some indicators the metadata tabs appear in this way:

![image](https://github.com/open-sdg/open-sdg/assets/117297044/0b32b4e1-d605-4286-9cf3-989491b34bea)

Also metadata information overflows

![image](https://github.com/open-sdg/open-sdg/assets/117297044/8170d8f3-4eed-47cb-96ce-59a350b11d6a)

Why does this happen?

Thank you

**Comment by brockfanning** [comment]
@SandraEliz This seems like maybe a bug. Can you also provide:
1. Operating system
2. Browser version
3. Link to page

**Comment by SandraEliz** [comment, marked as answer]
Hello

I don't know what happened with the tabs but they don't look bad anymore

And I solved the overflowing metadata by putting a style in site/_sass/custom.scss

td {
     overflow-wrap: anywhere;
}


The browser is:

![image](https://github.com/open-sdg/open-sdg/assets/117297044/09af620c-7af9-4495-87c6-998c03ff7a78)

OS
![image](https://github.com/open-sdg/open-sdg/assets/117297044/23d560f4-5f22-45f3-a934-b0c6611fd7e6)

Link:
https://prep-odsmexico.github.io/site/1-1-1-a/


Thank you
